slynk/backend/abcl.lisp

```diff
@@ -1370,27 +1361,27 @@
 (defmethod emacs-inspect ((object sys::structure-class))
   (let* ((name (jss::get-java-field object "name" t))
          (def (get name  'system::structure-definition)))
-  `((:label "Class: ") (:value ,object) (:newline)
```

Comment:
THis is very, extremely minor nit, and you may do this as you see fit, but I personally find it best to separate into commits indentation changes and funcional changes/fixes.  Here it seems you did a couple of the latter and mostly the former.  It could help the Gordon from the future 😉